side_runner_py/utils.py

```diff
@@ -22,13 +23,12 @@ def call_with_argname_bind(func, args_dict):
         if p.kind == Parameter.POSITIONAL_OR_KEYWORD
     ]
 
+    acceptable_args = {k: v for k, v in args_dict if k in param_names}
```

Comment:
maybe `args_dict` to `args_dict.items()`

Reply:
Fixed